src/Misc/layoutbin/en-US/strings.json

```diff
@@ -157,6 +157,8 @@
     "--auth                 Authentication type. Valid options are PAT (Personal Access Token),",
     "                       Negotiate (Kerberos or NTLM), Integrated (Windows default credentials) and",
     "                       ALT (Alternate Credentials)",
+    "--work                 Agent work folder, default to _work folder under agent root,",
```

Comment:
I don't know if it's worth mentioning that the `_work` folder should not be shared between agents (or can it)? I know it's probably obvious but who can tell what's in people's heads when they configure their things in a hurry.